version.cake

```diff
@@ -159,6 +164,47 @@ Task("StartNewVersion").Does(()=>
     ReplaceRegexInFilesWithExclusion("**/Info.plist", bundleShortVersionPattern, newBundleShortVersionString, "/bin/", "/obj/", "Demo");
 });
 
+// Fills sdk, android and ios versions in the build config file with the relevant ones.
+// sdkVersion must be provided as a parameter.
```

Comment:
We already have cake target to setup `sdkVersion`, please

1. Remove `sdkVersion` from there
2. Split to Android and iOS tasks
3. Add common empty task that depends on Android and iOS ones